hw/ip/otbn/dv/rig/rig/gens/small_val.py

```diff
@@ -62,7 +62,11 @@ def gen(self,
 
         # Pick a target value. For now, we take arbitrary values in the range
         # [-5, 5].
-        tgt_val = random.randint(-5, 5)
+        temp_val = random.randint(-64, 64)
+        if temp_val % 32:
+            tgt_val = temp_val << 5
```

Comment:
Looks fine, but could you update the comment above to match? (And explain why this is produces helpful values!)